pentaho/src/main/resources/solution/bi-developers/cda/template.html

```diff
@@ -211,8 +200,8 @@
     {
       type: "mondrian.jdbc (denormalized)",
       desc: "Denormalized Mdx query over a jdbc mondrian connection",
-      url: webAppPath + "/plugin/cda/api/doQuery?dataAccessId=1&path=/public/plugin-samples/cda/cdafiles/denormalized-mondrian-jndi.cda",
-      expectedCols: 5,
+      url: webAppPath + "/plugin/cda/api/doQuery?dataAccessId=1&path=/public/plugin-samples/cda/cdafiles/denormalized-mondrian-jdbc.cda",
+      expectedCols: 4,
```

Comment:
Yes, the "denormalized-mondrian-jndi.cda" was being called twice (for the jndi and for the jdbc) and the "denormalized-mondrian-jdbc.cda" was not being called.